python-regression/util/test_logic/value_fetch_logic.py

```diff
@@ -128,6 +128,16 @@ def fetch_bool_list(value):
         return [True] * len(response)
 
 
+def fetch_bool_list_mixed(value):
+    """
+    Returns a list filled with bool conversions of the input string separated by space".
+    :param value: The input value
+    :return: The list of bool values
+    """
+
+    list = value.split()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'list'](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=4682439)